src/UITests/UITests.csproj

```diff
@@ -80,15 +80,16 @@
       <HintPath>..\packages\Newtonsoft.Json.10.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
     </Reference>
     <Reference Include="Sarif, Version=2.0.0.0, Culture=neutral, PublicKeyToken=21a5e83f6f5bb844, processorArchitecture=MSIL">
-      <HintPath>..\packages\Axe.Windows.0.2.0-prerelease\lib\net471\Sarif.dll</HintPath>
+      <HintPath>..\packages\Sarif.Sdk.2.0.0-csd.2.beta.2018-10-10\lib\net461\Sarif.dll</HintPath>
```

Comment:
This as well (Sarif)

Reply:
Removed in iteration 5

---
In reply to: [303560850](https://github.com/microsoft/accessibility-insights-windows/pull/430#discussion_r303560850) [](ancestors = 303560850)